extensions/amp-list/0.1/amp-list.js

```diff
@@ -853,6 +856,9 @@ export class AmpList extends AMP.BaseElement {
         }).then(() => {
           this.attemptToFit_(dev().assertElement(this.container_));
         });
+      } else {
```

Comment:
Added this else clause so that if we don't have a `load-more-src` in the XHR, then we hide the see-more button and show the `load-ended` element if applicable.